todo/models.py

```diff
@@ -18,7 +18,14 @@ class Status(models.TextChoices):
     
     name = models.CharField(max_length=256)
     description = models.CharField(max_length=255)
-    status = models.CharField(max_length = 20, choices=Status.choices, default=Status.not_done)
+    status = models.CharField(max_length=20, choices=Status.choices, default=Status.not_done)
```

Comment:
```suggestion
    status = models.CharField(max_length=20, choices=Status.choices, default=Status.not_done, blank=True)
```

เพื่อที่ตอนสร้างไม่ต้องใส่ status มาก็ได้ ยังไง status เริ่มต้นก็เป็น not done